### ISSUE TITLE
Incorrect fieldset legend if value is django.utils.translation.ugettext_lazy

### ISSUE THREAD

**Issue by subuk** [earle/django-bootstrap#41]
Fix:

```
diff --git a/bootstrap/forms.py b/bootstrap/forms.py
index 988e5e4..7c3f4a7 100755
--- a/bootstrap/forms.py
+++ b/bootstrap/forms.py
@@ -160,11 +160,11 @@ class Fieldset(object):
     """ Fieldset container. Renders to a <fieldset>. """

     def __init__(self, legend, *fields, **kwargs):
-        self.legend_html = legend and ('<legend>%s</legend>' % legend) or ''
+        self.legend_html = legend and legend or ''
         self.fields = fields
         self.css_class = kwargs.get('css_class')

     def as_html(self, form):
         class_str = self.css_class and (' class="%s"' % self.css_class) or ''
-        return u'<fieldset%s>%s%s</fieldset>' %  (class_str, self.legend_html, form.render_fields(self.fields), )
+        return u'<fieldset%s><legend>%s</legend>%s</fieldset>' %  (class_str, unicode(self.legend_html), form.render_fields(self.fields), )
```

**Comment by subuk** [comment]
Without this fix fieldset legend rendered as

`<django.utils.functional.__proxy__ object at ...>`

**Comment by pthrasher** [comment]
fixed by pull request #54
